lib/src/generator/generator.dart

```diff
@@ -20,6 +21,10 @@ String generateFile(Resources res) {
     }
   }
 
+  if (arguments.assetClasses?.isNotEmpty == true) {
+    fullCode += "import 'asset_classes.dart';";
```

Comment:
can we get around of hardcoding the asset classes file by fully specify the import in the `pubspec.yaml`
maybe by specify it like `asset_classes.dart#SvgFile` or splitting it in multiple lines:
```yaml
".svg":
    import: asset_classes.dart
    class: SvgFile
```
it could fallback to `asset_classes.dart` when no import is defined
this would make the usage more flexible and would also make it possible to  use classes from other packages directly.